mlflow/protos/service.proto

```diff
@@ -78,6 +78,10 @@ service MlflowService {
   rpc searchExperiments (SearchExperiments) returns (SearchExperiments.Response) {
     option (rpc) = {
       endpoints: [{
+        method: "POST",
+        path: "/mlflow/experiments/search"
+        since { major: 2, minor: 0 },
+      }, {
         method: "GET",
         path: "/mlflow/experiments/search"
         since { major: 2, minor: 0 },
```

Comment:
Preserving `GET` for consistency with Databricks backend protos; `POST` will be preferentially used since it is listed first.